tests/ui-toml/large_include_file/clippy.toml

```
@@ -0,0 +1 @@
+max-include-file-size = 400
```

Could you add a line break here?
```suggestion
max-include-file-size = 400

```